process_manager/runner.py

```diff
@@ -0,0 +1,124 @@
+
+import dataclasses
+import functools
+import logging
+from itertools import count
+
+from process import ProcessTask, Result
+from multiprocessing import Process
+
+job_counter = count()
+
+@dataclasses.dataclass(order=True)
+class Job:
+    process: ProcessTask = dataclasses.field(compare=False)
+    sequence: int = dataclasses.field(compare=False,
```

Comment:
Maybe this is something I don't know about `dataclass`, but if you mark both fields as `compare=False`, what does `order` compare on?